commitlint.config.js

```diff
@@ -0,0 +1 @@
+module.exports = { extends: ['@commitlint/config-conventional'] };
```

Comment:
Let's start the message with first letter in upper case.

Then, add this rule

```suggestion
module.exports = {
  extends: ['@commitlint/config-conventional'],
  rules: {
    'subject-case': [2, 'always', 'sentence-case'],
  }
};
```